packages/cli/src/lib/graphql/business-platform/active-shops.ts

```diff
@@ -0,0 +1,120 @@
+import {businessPlatformRequest} from '@shopify/cli-kit/node/api/business-platform';
+
+const CurrentUserAccountQuery = `#graphql
+  query currentUserAccount {
+    currentUserAccount {
+      organizations(first: 100) {
```

Comment:
I'm not handling pagination at the moment. Showing many shops might not look nice in the terminal... perhaps we should just add a select choice at the end for "Can't see my shop in this list", and then prompt a text input?

Reply:
Actually, I just remembered there's a `renderAutocompletePrompt` that might be nice for this...